eng/Packages.Data.props

```diff
@@ -272,6 +272,7 @@
     <PackageReference Update="ApprovalTests" Version="3.0.22" />
     <PackageReference Update="ApprovalUtilities" Version="3.0.22" />
     <PackageReference Update="Azure.Core" Version="1.44.1" />
+    <PackageReference Update="Azure.ResourceManager" Version="1.13.0" />
```

Comment:
Please move this down to preserve alphabetical ordering

Reply:
updated